src/components/adaptive-cards/Component/Component.jsx

```diff
@@ -132,12 +135,19 @@ export default function Component({data}) {
   return (
     <C
       data={data}
-      className={classes.join(' ')}
-      style={style}
+      className={`${cssClasses} ${classes.join(' ')}`}
+      style={{...styleProp, ...style}}
```

Comment:
in order to let developers overwrite some styles if they wish, let's put styleProp after style